fuelclient/cli/actions/network.py

```diff
@@ -12,10 +12,26 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
+import netaddr
+from fuelclient.cli import error
 from fuelclient.cli.actions.base import Action
 import fuelclient.cli.arguments as Args
 from fuelclient.cli.arguments import group
 from fuelclient.objects.environment import Environment
+from fuelclient.objects.plugins import master_only
```

Comment:
`master_only` could be now move to some more general use file (utils? decorators?)